src/utils/tablify.js

```diff
@@ -33,11 +33,24 @@ const chars = {
 	middle: '│',
 };
 
+const errorCyclicObject = 'Error: cyclic object cannot be displayed.';
```

Comment:
`const ERROR_CYCLIC_OBJECT = 'Error: Cyclic object cannot be displayed in a table.'`

Reply:
Is this a good approach? We're not throwing anything after all, so it's weird to describe it as an error. Why not use `[Circular]` like Node does?

Reply:
I think for users, it's good to show maximum things we can show (all the information user want might already be there) and show there were more information but `circular`.
Instead of `Error`, but maybe `Warning`? 

As for the name I took it from 
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cyclic_object_value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cyclic_object_value)

Reply:
I still think it's weird to describe this as an error if nothing gets thrown. It's really just how we display circular structures.